File: app/src/main/java/com/vn/ezlearn/modelresult/ExamsResult.java
package com.vn.ezlearn.modelresult;

import com.vn.ezlearn.models.Category;
import com.vn.ezlearn.models.Exam;

import java.util.List;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 05/10/2017.
 */

public class ExamsResult {
    public Boolean success;
    public ExamsData data;

    public static class ExamsData {
        public Category category;
        public List<Exam> list;
        public Integer totalCount;
    }
}

File: app/src/main/java/com/vn/ezlearn/modelresult/ExamsResult.kt
package com.vn.ezlearn.modelresult

import com.vn.ezlearn.models.Category
import com.vn.ezlearn.models.Exam

/**
 * Created by FRAMGIA\nguyen.duc.manh on 05/10/2017.
 */

class ExamsResult {
    var success: Boolean = false
    var data: ExamsData? = null

    class ExamsData {
        var category: Category? = null
        var list: List<Exam>? = null
        var totalCount: Int = 0
    }
}
